src/node/nodestate.h

```diff
@@ -266,8 +258,8 @@ namespace ccf
       self = 0;
       setup_pbft();
       setup_history();
+      consensus->force_become_leader();
```

Comment:
Why is this necessary here?

Reply:
Because frontend will check to see if we are the leader, see that we aren't and try to redirect the request to the leader, but there won't be one.